tests/data/test_data_generator.py

```diff
@@ -0,0 +1,54 @@
+import os
+import numpy as np
+import pandas as pd
+from skmultiflow.data import DataGenerator
+
+
+def test_data_stream_X_y(test_path, package_path):
+    test_file = os.path.join(package_path, 'src/skmultiflow/data/datasets/sea_stream.csv')
```

Comment:
Is it reasonable to split this into multiple tests?

Reply:
Done